tests/fixtures-valid.js

```diff
@@ -27,22 +53,60 @@ const uniqueValues = {
 };
 
 const promises = [];
-
-// search fixture files
 const fixturePath = path.join(__dirname, `..`, `fixtures`);
-for (const manKey of fs.readdirSync(fixturePath)) {
-  const manDir = path.join(fixturePath, manKey);
-
-  // files in manufacturer directory
-  if (fs.statSync(manDir).isDirectory()) {
-    for (const file of fs.readdirSync(manDir)) {
-      if (path.extname(file) === `.json`) {
-        const fixKey = path.basename(file, `.json`);
-        handleFixtureFile(manKey, fixKey);
+
+
+if (args.fixtures !== undefined) {
+  // were assuming the fixtures are given as a semicolon
+  // separated list in the form `<manufacturer>[/[fixtures]]`
+  // where [fixtures] is a comma separated list of concrete fixtures
+
+  const fixturesToParse = args.fixtures;
+  const allManufacturers = fs.readdirSync(fixturePath);
+  for (const manufacturerFixtures of fixturesToParse.split(`;`)) {
+    let [manKey, fixtures] = manufacturerFixtures.split(`/`);
+    if (!allManufacturers.includes(manKey)) {
+      promises.push(new Promise(resolve => {
+        resolve({
+          filename: manKey,
+          errors: [`invalid manufacturer`]
```

Comment:
```suggestion
          errors: [`invalid manufacturer`],
          warnings: []
```

`warnings` must be specified as well (there's a call to `result.warnings.length` in the end of the script). I have to admit that this result object has been poorly documented, though. (But the needed refactoring is a different task.)